web3js-ext/test/role_based_key_feepayer.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { Web3, TxType, AccountKeyType, getPublicKeyFromPrivate, toPeb } from "@kaiachain/web3js-ext";
+import { assert } from "chai";
+
+const provider = new Web3.providers.HttpProvider("https://public-en-kairos.node.kaia.io");
+const web3 = new Web3(provider);
+
+type Account = {
+    address: string;
+    privateKey: string;
+    signTransaction: (tx: any) => Promise<any>;
+}
+
+// Feedback1. Generate Temporary Key.
+function generateTemporaryAccount(): Account {
+    return web3.eth.accounts.create();
+}
+
+describe("Role-based Key Tests", function () {
+    this.timeout(10000);
+
+    let roleTransactionAccount: Account;
+    let roleAccountUpdate: Account;
+    let roleFeePayerAccount: Account;
+
+    // Before all tests, set up Role-based Key
+    before(async function () {
+        console.log("\n--- Generating Temporary Accounts ---");
+        roleTransactionAccount = generateTemporaryAccount();
+        roleAccountUpdate = generateTemporaryAccount();
+        roleFeePayerAccount = generateTemporaryAccount();
```

Comment:
[[Feeback1]](https://github.com/kaiachain/kaia-sdk/pull/59/files#r1843196541) fixed this test to generate temporary key and update the account key to role based.
cc. @kjeom